app/jobs/export_cocina_json_job.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+##
+# A job that exports line-oriented JSON for one or more objects
+# @param [Integer] bulk_action_id GlobalID for a BulkAction object
+# @param [Hash] params additional parameters that an Argo job may need
```

Comment:
We might want these two lines immediately above `def perform` rather than on the class def